scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -156,8 +156,8 @@ task ExtractTask {
     # Runtime settings:
     runtime {
         docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_d8a72b825eab2d979c8877448c0ca948fd9b34c7_change_to_hwe"
-        memory: "10 GB"
-        disks: "local-disk 100 HDD"
+        memory: "12 GB"
```

Comment:
(just a nice-to-have) does it make sense to document our rationale for these choices somewhere? even a distinction between "this value was chosen deliberately" and "this is an inherited value and we haven't thought about this much" could be useful in future?